src/main/java/io/xpring/javascript/JavaScriptUtils.java

```diff
@@ -31,6 +32,41 @@ public boolean isValidAddress(String address) {
         return isValidAddressFunction.execute(address).asBoolean();
     }
 
+    /**
+     * Encode the given classic address and tag into an X-Address.
+     *
+     * @param classicAddress A classic address to encode.
+     * @param tag An optional tag to encode.
+     * @return A new X-Address if inputs were valid, otherwise null.
+     * @see <a href="https://xrpaddress.info/">https://xrpaddress.info/</a>
+     */
+    public String encodeXAddress(String classicAddress, Long tag) {
+        Value encodeXAddressFunction = javaScriptUtils.getMember("encodeXAddress");
+        Value result = tag != null ? encodeXAddressFunction.execute(classicAddress, tag) : encodeXAddressFunction.execute(classicAddress);
+        return result.asString();
+    }
+
+    /**
+     * Decode a {@link ClassicAddress} from a given X-Address.
+     *
+     * @param xAddress The xAddress to decode.
+     * @return A {@link ClassicAddress} if the inputs were valid, otherwise null.
+     * @see <a href="https://xrpaddress.info/">https://xrpaddress.info/</a>
+     */
+    public ClassicAddress decodeXAddress(String xAddress) {
```

Comment:
If `null` isn't a valid argument it may be worth using Objects.requireNonNull(xAddress) here.

Reply:
Done